### DISCUSSION TITLE
the sample of FV3 outputs for UFS_IC_generator.py to create the forcing

### DISCUSSION THREAD

**Discussion by HelinWei-NOAA** [NCAR/ccpp-scm#266] [Cases, Initial Conditions, Forcing]
The current UFS_IC_generator.py only creates ICs and set all forcing to 0. Now I would like to create the  real forcing from UFS. What kind of output can be used directly by this script? Or this script has to be modified?

**Comment by grantfirl** [comment]
Helin,

Yes, the version that is in the main branch is limited to providing initial conditions only. There is a branch with an updated version of this script that has forcing, although it isn't necessarily ready to be released to the public since there may be some bugs: https://github.com/NCAR/ccpp-scm/pull/224. You're welcome to try to use that. I wrote some documentation for it here: https://docs.google.com/document/d/18LISyhHVBDmKYcMefYfypHNcnEPQ9hOq4XKd6XvulGs/edit?usp=sharing

Unfortunately, due to competing priorities, I won't be able to finish this work and commit it to the main branch until later this Summer.

-Grant

**Comment by HelinWei-NOAA** [comment]
Grant,

Thank you so much. This is really helpful. I will try to test it first. We
have a summer intern who is going to use SCM to investigate the cold biases
under the stable boundary conditions. He will also help to get this working.

Helin
